COPYRIGHT

```diff
@@ -0,0 +1,23 @@
+Intellectual Property Notice
+------------------------------
+
+Apollo is licensed under the Apache License, Version 2.0 (LICENSE-APACHE
+or http://www.apache.org/licenses/LICENSE-2.0) or the MIT license,
+(LICENSE-MIT or http://opensource.org/licenses/MIT), at your option.
+
+Copyrights and patents in the Apollo project are retained by contributors.
+No copyright assignment is required to contribute to Apollo.
```

Comment:
We cannot just switch the license of Apollo from `MIT` to `Apache-2.0 OR MIT`.  It has to stay MIT, unless we want to send it through the process again AND get all contributors' permission, so please change all these `Apache-2.0 OR MIT`'s to `MIT`.

For a single-licensed project, we do't need all this fanciness so also remove `COPYRIGHT`.